worlds/hk/__init__.py

```diff
@@ -684,42 +721,7 @@ def _hk_notches(self, player: int, *notches: int) -> int:
         return sum(self.multiworld.worlds[player].charm_costs[notch] for notch in notches)
 
     def _hk_option(self, player: int, option_name: str) -> int:
-        return getattr(self.multiworld, option_name)[player].value
+        return getattr(self.multiworld.worlds[player].options, option_name).value
```

Comment:
Curious what the blocker is for the remaining ones to be removed?

Reply:
the ones i moved are only used in setting completion condition, I just didn't want to go through the effort to confirm the other ones can be properly moved out of logicmixin (yet)